### ISSUE TITLE
Adjustable as a container property

### ISSUE THREAD

**Issue by akaDuality** [VODGroup/VoiceOverDesigner#269]
We should have different behaviour for VoiceOver and Voice Control for adjustable elements.

For Voice Control we should split button to communicate with app like "Press M" (or press Middle, when we have accessibilityInputLabels) to select middle size.
<img width="614" alt="Screenshot 2023-09-24 at 14 31 13" src="https://github.com/VODGroup/VoiceOverDesigner/assets/3120680/6637003b-8f8c-4aac-8380-34a7c9feec11">

For VoiceControl we should wrap in adjustable elements: focus on single element, but change value by vertical swipe.
<img width="1016" alt="Screenshot 2023-09-24 at 14 32 31" src="https://github.com/VODGroup/VoiceOverDesigner/assets/3120680/400f7e8f-389a-4ca4-8e1a-2fa2806dc8a8">

We can join both behaviours by adding "adjustable" type for containers. The property can provide elements' labels like a values of adjustable control.
